src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -2004,5 +2020,130 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_Microsoft_PowerShell_Co
+                        .AddItemScriptBlock(@"""$($_.Foreground.LightGreen)Example Text$($PSStyle.Reset)""", label: "Foreground.LightGreen")
+                        .AddItemScriptBlock(@"""$($_.Foreground.Yellow)Example Text$($PSStyle.Reset)""", label: "Foreground.Yellow")
+                        .AddItemScriptBlock(@"""$($_.Foreground.LightYellow)Example Text$($PSStyle.Reset)""", label: "Foreground.LightYellow")
+                        .AddItemScriptBlock(@"""$($_.Background.Black)Example Text$($PSStyle.Reset)""", label: "Background.Black")
+                        .AddItemScriptBlock(@"""$($_.Background.White)Example Text$($PSStyle.Reset)""", label: "Background.White")
+                        .AddItemScriptBlock(@"""$($_.Background.DarkGray)Example Text$($PSStyle.Reset)""", label: "Background.DarkGray")
+                        .AddItemScriptBlock(@"""$($_.Background.LightGray)Example Text$($PSStyle.Reset)""", label: "Background.LightGray")
+                        .AddItemScriptBlock(@"""$($_.Background.Red)Example Text$($PSStyle.Reset)""", label: "Background.Red")
+                        .AddItemScriptBlock(@"""$($_.Background.LightRed)Example Text$($PSStyle.Reset)""", label: "Background.LightRed")
+                        .AddItemScriptBlock(@"""$($_.Background.Magenta)Example Text$($PSStyle.Reset)""", label: "Background.Magenta")
+                        .AddItemScriptBlock(@"""$($_.Background.LightMagenta)Example Text$($PSStyle.Reset)""", label: "Background.LightMagenta")
+                        .AddItemScriptBlock(@"""$($_.Background.Blue)Example Text$($PSStyle.Reset)""", label: "Background.Blue")
+                        .AddItemScriptBlock(@"""$($_.Background.LightBlue)Example Text$($PSStyle.Reset)""", label: "Background.LightBlue")
+                        .AddItemScriptBlock(@"""$($_.Background.Cyan)Example Text$($PSStyle.Reset)""", label: "Background.Cyan")
+                        .AddItemScriptBlock(@"""$($_.Background.LightCyan)Example Text$($PSStyle.Reset)""", label: "Background.LightCyan")
+                        .AddItemScriptBlock(@"""$($_.Background.Green)Example Text$($PSStyle.Reset)""", label: "Background.Green")
+                        .AddItemScriptBlock(@"""$($_.Background.LightGreen)Example Text$($PSStyle.Reset)""", label: "Background.LightGreen")
+                        .AddItemScriptBlock(@"""$($_.Background.Yellow)Example Text$($PSStyle.Reset)""", label: "Background.Yellow")
+                        .AddItemScriptBlock(@"""$($_.Background.LightYellow)Example Text$($PSStyle.Reset)""", label: "Background.LightYellow")
+                        .EndEntry()
+                .EndList());
+        }
+
+        private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Automation_PSStyleFormattingData()
+        {
+            yield return new FormatViewDefinition("System.Management.Automation.PSStyle+FormattingData",
```

Comment:
The formatting system currently has a problem that `Format-List` leak the ANSI settings of a property value.

![image](https://user-images.githubusercontent.com/127450/100816316-af404f00-33fa-11eb-9896-77e983ff9f63.png)

Reply:
After this PR gets merged, I intend to update Format-List and Format-Table, but for generalized usage, you won't see the escape sequence definition and what will happen is you'll just see the labels but no value, but will properly reset attributes.